requests/database/hasOpticalData.graphql

```diff
@@ -0,0 +1,15 @@
+query {
+  byComponentId: hasOpticalData(
```

Comment:
Just a note: The `has*Data` queries are implemented by now in the reference implementation. They are just not deployed yet.